kivymd/uix/button/button.py

```diff
@@ -659,93 +656,127 @@ class BaseButton(ThemableBehavior, ButtonBehavior, AnchorLayout):
     """
 
     _radius = NumericProperty(0)
-    _md_bg_color = ColorProperty(None)  # last current button color
+    _min_width = NumericProperty(None)
+    _md_bg_color = ColorProperty(None)
+    _md_bg_color_disabled = ColorProperty(None)
+    _line_color = ColorProperty(None)
+    _line_color_disabled = ColorProperty(None)
+    _theme_text_color = OptionProperty(
+        None,
+        options=[
+            "Primary",
+            "Secondary",
+            "Hint",
+            "Error",
+            "Custom",
+            "ContrastParentBackground",
+        ],
+    )
+    _text_color = ColorProperty(None)
+
+    # Default colors - set to None to use primary theme colors
+    _default_md_bg_color = [0.0, 0.0, 0.0, 0.0]
+    _default_md_bg_color_disabled = [0.0, 0.0, 0.0, 0.0]
+    _default_line_color = [0.0, 0.0, 0.0, 0.0]
+    _default_line_color_disabled = [0.0, 0.0, 0.0, 0.0]
+    _default_theme_text_color = StringProperty('Primary')
+    _default_text_color = ColorProperty(None)
+
+    _animation_fade_bg = ObjectProperty(None, allownone=True)
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        if not self.md_bg_color_disabled:
-            self.md_bg_color_disabled = self.theme_cls.disabled_hint_text_color
-        self.theme_cls.bind(primary_palette=self.update_md_bg_color)
```

Comment:
iirc, we can just use
```py
self.theme_cls.bind(var1=self.update_var1, var2=self.update_var2)
```
It's not big deal, i think

Nice Work :)

Reply:
I have combined bind calls (where possible) :)